src/Framework/TestCase.php

```diff
@@ -2625,4 +2631,24 @@ private function setDoesNotPerformAssertionsFromAnnotation()
             $this->doesNotPerformAssertions = true;
         }
     }
+
+    /**
+     * @param ReflectionObject $reflection
+     * @return bool
+     */
+    private function isSaveCloneForMockObjectsFromTestArguments(ReflectionObject $reflection)
+    {
+        if (method_exists($reflection, 'isCloneable') && !$reflection->isCloneable()) {
```

Comment:
This is not needed. `ReflectionClass::isCloneable()` was introduced in PHP 5.4 and PHPUnit 5 requires PHP 5.6.